libraries/AP_InertialSensor/AP_InertialSensor_Invensense.cpp

```diff
@@ -618,6 +621,7 @@ bool AP_InertialSensor_Invensense::_accumulate_fast_sampling(uint8_t *samples, u
 
         _accum.gyro += _accum.gyro_filter.apply(g);
         _accum.count++;
+        _seqcnt++;
```

Comment:
Is this really supposed to be here? It means we'll have gaps in sequence numbers. I thought this was for logging how many samples did the InertialSensor library had published, but this is logging how many samples a driver has received from the sensor - if that's what is intended disregard this comment.